Task Liverpool/src/test/java/com/encora/automation/test/HomeTest.java

```diff
@@ -1,12 +1,33 @@
 package com.encora.automation.test;
 
+import org.testng.Assert;
+import pages.HomePage;
+import pages.ProductPage;
 import org.testng.annotations.Test;
 
 public class HomeTest extends BaseTest{
 
+    private HomePage homePage;
+    private ProductPage productPage;
+
+    private HomeTest(){
+        homePage = new HomePage(getDriver());
```

Comment:
is not needed to inizialice all Pages, only needed (for this test is OK but considere for future implementations)


If you need to inizialice something for the Test, I would recomment to use an `@BeforeTest` anotation

Reply:
ok.